.circleci/config.yml

```diff
@@ -0,0 +1,178 @@
+version: '2.1'
+orbs:
+  gh: circleci/github-cli@2.1.1
+  maven: circleci/maven@1.4.0
+  node: circleci/node@5.0
+  slack: circleci/slack@4.12.0
+
+executors:
+  mvn-11:
+    docker:
+      - image: 'cimg/openjdk:11.0'
+
+commands:
+  notify-slack:
+    description: "Notify slack on channel << parameters.channel >>"
+    parameters:
+      channel:
+        type: string
+      mentions_failure:
+        type: string
+        default: ''
+      mentions_success:
+        type: string
+        default: ''
+    steps:
+      - slack/notify:
+          event: fail
+          channel: << parameters.channel >>
+          mentions: << parameters.mentions_failure >>
+          template: basic_fail_1
+      - slack/notify:
+          event: pass
+          channel: << parameters.channel >>
+          mentions: << parameters.mentions_success >>
+          template: basic_success_1
+  push-artifacts:
+    description: | 
+      Places the artifacts in in the given repository, to trigger the target pipeline.
+      If the branch is "dev", it will push straight away, if it's trino-snowflake-connector, it will raise a PR.
+    parameters:
+      target_repository:
+        type: string
+      target_branch:
+        type: enum
+        enum: ["dev", "trino-snowflake-connector"]
```

Comment:
Does this need to be an enum (i.e. constrained to set of allowed values)?

Reply:
it doesn't need to, but given that it's all self-contained in the same yml file, we can make use of yaml syntax validation, just to avoid typos